docs/python-api/python-semantics.rst

```diff
@@ -0,0 +1,36 @@
+Triton mostly follows the semantics of NumPy with minor exceptions. In this document, we go over some of the array computing features supported in Triton, and we cover the exceptions where Triton's semantics deviate from that NumPy.
```

Comment:
This might be a good place to also mention that division and module follow the C definition and not Python's.

Reply:
Oh an also that constexpr values _do_ follow python semantics. This can be a bit confusing as the meaning of `a // b` actually changes whether `a` and `b` are both constexpr.

Reply:
Ah, I forgot to write that section! Div and mod is exactly why I wrote this in the first place...